presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -199,6 +199,7 @@
     public static final String VERBOSE_EXCEEDED_MEMORY_LIMIT_ERRORS_ENABLED = "verbose_exceeded_memory_limit_errors_enabled";
     public static final String MATERIALIZED_VIEW_DATA_CONSISTENCY_ENABLED = "materialized_view_data_consistency_enabled";
     public static final String QUERY_OPTIMIZATION_WITH_MATERIALIZED_VIEW_ENABLED = "query_optimization_with_materialized_view_enabled";
+    public static final String AGG_IF_TO_FILTER_REWRITE_ENABLED = "agg_if_to_filter_rewrite_enabled";
```

Comment:
AGGREGATION_IF_TO_FILTER_REWRITE_ENABLED; let's spell out "aggregation" in this PR